relay-common/src/glob.rs

```diff
@@ -77,7 +77,9 @@ fn translate_codeowners_pattern(pattern: &str) -> Option<Regex> {
     for (i, ch) in iterator {
         if let Some(skip_amount) = num_to_skip {
             num_to_skip = Some(skip_amount - 1);
-            continue;
+            if num_to_skip > Some(0) {
```

Comment:
This prevents everything after the `*` in the pattern from being skipped

Reply:
Can you add this as a comment directly in the code?